app/controllers/api/tags_controller.rb

```diff
@@ -42,7 +42,8 @@ def delete_resource(_type, id, _data = {})
     private
 
     def fetch_category(data)
-      category_id = parse_id(data, :categories) || parse_by_attr(data, :categories, %w(name))
+      category_id = parse_id(data, :categories)
+      category_id ||= collection_class(:categories).find_by_name(data["name"]).id if data["name"]
```

Comment:
It will - I considered using `try` here, but that would change the behavior - unless I'm mistaken this should behave exactly as before, and would rather make behavioral changes in a separate PR (with tests)